manifests/reset-search-default-extension.yml

```diff
@@ -0,0 +1,9 @@
+description: Extension asks users to reset their default search for some extensions
+repo-prefix: searchdefaultaddon
+active: true
+private-repo: true
+branch: main
+artifacts:
+  - web-ext-artifacts/reset-search-defaults.xpi
+addon-type: privileged
```

Comment:
I think this was `system`: https://github.com/mozilla-extensions/xpi-manifest/blob/master/xpi-manifest.yml#L240

Reply:
yes it was changed recently